CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/ImportTableColumn.java

```diff
@@ -0,0 +1,152 @@
+            final AttributeTranslator parser = attributeNode.getTranslator();
+            final PluginParameters parserParameters = attributeNode.getTranslatorParameters();
+            final String defaultValue = attributeNode.getDefaultValue();
+
+            Class<? extends AttributeDescription> attributeDescriptionClass = AttributeRegistry.getDefault().getAttributes().get(attributeNode.getAttribute().getAttributeType());
+            // Handle pseudo-attributes
+            if (attributeDescriptionClass == null && attributeNode.getAttribute().getName().equals(ImportController.DIRECTED)) {
+                attributeDescriptionClass = BooleanAttributeDescription.class;
+            }
+            try {
+                final AttributeDescription attributeDescription = attributeDescriptionClass.getDeclaredConstructor().newInstance();
+                for (final TableRow row : data) {
+                    final CellValueProperty property = row.getProperty(columnIndex);
+                    final String value = property.get().getOriginalText();
+                    final String parsedValue = parser.translate(value, parserParameters);
+                    final String errorMessage = attributeDescription.acceptsString(parsedValue);
+                    columnFailed |= errorMessage != null;
+                    if (parsedValue == null ? value == null : parsedValue.equals(value)) {
+                        property.setText(value);
+                        if (errorMessage != null && defaultValue == null) {
+                            property.setMessage(errorMessage, true);
+                        } else {
+                            property.setMessage(null, false);
+                            columnFailed = false;
+                        }
+                    } else {
+                        property.setText(parsedValue);
+                        if (errorMessage != null) {
+                            property.setMessage(errorMessage, true);
+                        } else {
+                            property.setMessage("Original: " + value, false);
+                        }
+                    }
+                }
+            } catch (final IllegalAccessException | IllegalArgumentException
+                    | InstantiationException | NoSuchMethodException
+                    | SecurityException | InvocationTargetException ex) {
+                LOGGER.log(Level.SEVERE, ex.getLocalizedMessage(), ex);
+            }
+        } else {
+            for (final TableRow row : data) {
+                final CellValueProperty property = row.getProperty(columnIndex);
+                property.setText(property.get().getOriginalText());
+                property.setMessage(null, false);
+            }
+        }
+
+        if (columnFailed) {
+            getGraphic().setStyle("-fx-background-color: rgba(255, 0, 0, 0.3);");
+        } else {
+            getGraphic().setStyle("-fx-background-color: transparent;");
+        }
```

Comment:
can convert this to a tertiary statement: `getGraphic().setStyle(columnFailed ? .. : ...)`